src/main/java/gyro/google/GoogleResource.java

```diff
@@ -114,6 +115,12 @@ protected static String formatGoogleExceptionMessage(GoogleJsonResponseException
             .collect(Collectors.joining("\n"));
     }
 
+    protected static String formatOperationErrorMessage(Operation operation) {
+        return operation.getError().getErrors().stream()
+            .map(Operation.Error.Errors::getMessage)
+            .collect(Collectors.joining("\n"));
+    }
+
```

Comment:
As this operation is part of the compute model, this should be part of `ComputeResource`.
Also better to use `Operation.Error` as the parameter, to avoid NPE if someone called this method without doing a null check